include/solvers/heat_transfer_scratch_data.h

```diff
@@ -427,9 +424,9 @@ class HeatTransferScratchData
    */
   FEValuesExtractors::Vector velocities;
   // This FEValues must mandatorily be instantiated for the velocity
-  FEValues<dim>               fe_values_navier_stokes;
-  std::vector<Tensor<1, dim>> velocity_values;
-  std::vector<Tensor<2, dim>> velocity_gradient_values;
+  FEValues<dim>               fe_values_fd;
+  std::vector<Tensor<1, dim>> velocity_values_fd;
```

Comment:
yes @CaroleAnneDaunais said the same, I was not sure myself about that but thought that maybe it would be clearer. As you both say it is not, I will revert this :)